ui/src/app/core/model/model.service.ts

```diff
@@ -39,6 +39,11 @@ export class ModelService {
     return this.http.put<Model>(`${environment.adminApiUrl}app/${applicationId}/model/${modelId}`, { name });
   }
 
+  public clone(applicationId: string, modelId: string, name: string): Observable<Model> {
```

Comment:
please remove public - according to angular styleguide it is redundant

Reply:
Ok, fixed